scripts/merge_counts.R

```diff
@@ -0,0 +1,28 @@
+# Author: Ioannis Moustakas, i.moustakas@lumc.nl (Based on a script by Szymon Kielbasa)
```

Comment:
puting this a separated file will I think not work. Better to use this kind of construct:
```
command<<<
  python <<CODE
    with open("${in}") as fp:
      for line in fp:
        if not line.startswith('#'):
          print(line.strip())
  CODE
  >>>
```

See also: https://github.com/openwdl/wdl/blob/master/SPEC.md#alternative-heredoc-syntax

Of course changing python to Rscript and replace the code into the wdl file itself. Also the argparser will not be needed then because you can use wdl variables inside the Rscript (not sure about the listOfFiles ;) )